charts/cassandra-migrations/templates/galley-migrate-data.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: galley-migrate-data
```

Comment:
Which data does this migrate? Maybe it's nice to add a comment / link to the executable that does something here.

Reply:
I will add the link once the wire-server PR gets merged.

Reply:
This is done now.